docs/qcportal/source/install.rst

```diff
@@ -1,39 +1,64 @@
 Install QCPortal
 =================
 
-You can install ``qcportal`` with ``conda``, with ``pip``, or by installing from source.
+You can install ``qcportal`` with ``conda`` or with ``pip``.
 
 Conda
 -----
 
-You can install or update ``qcportal`` using `conda <https://www.anaconda.com/download/>`_::
+You can install qcportal using `conda <https://www.anaconda.com/download/>`_:
 
-    conda install qcportal -c conda-forge
+.. code-block:: console
 
-The ``qcportal`` package is maintained on the
+    >>> conda install qcportal -c conda-forge
+
+This installs QCPortal and its dependencies. The qcportal package is maintained on the
 `conda-forge channel <https://conda-forge.github.io/>`_.
 
 
 Pip
 ---
 
-To install or update ``qcportal`` with ``pip``::
+you can also install QCPortal using ``pip``:
+
+.. code-block:: console
+
+   >>> pip install qcportal
+
+
+Test the Installation
+---------------------
+
+.. note::
+
+   QCPortal can interface with `Plotly <https://plot.ly/>`_ for additional visualization features.
+   This is not installed by default and so you can expect the visualization tests to all be marked with
```

Comment:
It is installed with conda by default?

Reply:
The tests were still skipped, and I see what happened here. The `S22Fixture` is currently set to always skip.

https://github.com/MolSSI/QCFractal/blob/master/qcfractal/interface/tests/test_visualization.py#L19-L26

Reply:
But skip tests and plotly not installed are different?

https://github.com/conda-forge/qcportal-feedstock/blob/master/recipe/meta.yaml#L32

Reply:
Yes. And I'm currently changing what I wrote here. I just saw all the viz. stuff get skipped in my testing and incorrectly assumed since they all skipped it was from a missing package similar to services in Fractal and qc codes in Engine. 

Do you want me to turn back on the `S22Fixture` now that we have upgraded everything while I am looking at it anyways?